click/js/app.js

```diff
@@ -2,4 +2,21 @@ console.log("Sanity Check: JS is working!");
 
 $(document).ready(function() {
 
+  $(".span").on("click", function (events){
+
+    $("span").on("click", function handleClick(event) {
+    var itemText = "You Clicked: " + $(this).text() + " at " + Date.now();
+    $("ul").append("<li>" + itemText + "</li>");
+  });
+  })
 });
+
+//my original code
+$(".span").on("click", function (events){
+  $("ul").append("events");
+}
+
```

Comment:
Thanks for including your original code.  "This" will click with time.  Nathan likes to say "this" is "me, the thing you clicked (or hovered, or whatever...). "